spec/index.bs

```diff
@@ -594,8 +594,6 @@ Command Recording {#command-recording}
 
 <script type=idl>
 interface GPUProgrammablePassEncoder : GPUObjectBase {
-    void endPass();
-
     // Allowed in both compute and render passes
     void setBindGroup(u32 index, GPUBindGroup bindGroup, optional sequence<u64> dynamicOffsets);
```

Comment:
I just noticed that there is no `setBindGroup` during Bundle recording. Is this intentional?

Reply:
It should be in Bundle recording: `GPURenderBundleEncoder` is a `GPURenderPassBaseEncoder` which is a `GPUProgrammablePassEncoder`.

Reply:
oh I see. Got confused because of the naming inconsistency. There is a `GPURenderPassEncoder` versus `GPURenderBundleEncoder`, which kinda puts "pass" versus "bundle" in the context of encoding